src/main/webapp/app/exam/manage/exams/exam-checklist-component/exam-checklist.component.ts

```diff
@@ -61,10 +61,12 @@ export class ExamChecklistComponent implements OnInit {
         // calculate mandatory points and optional points
         if (this.pointsExercisesEqual) {
             this.exam.exerciseGroups!.forEach((exerciseGroup) => {
-                if (exerciseGroup.isMandatory) {
-                    sumPointsExerciseGroupsMandatory += exerciseGroup!.exercises![0]!.maxPoints!;
-                } else {
-                    sumPointsExerciseGroupsOptional += exerciseGroup!.exercises![0]!.maxPoints!;
+                if (exerciseGroup!.exercises && exerciseGroup.exercises.length !== 0) {
+                    if (exerciseGroup.isMandatory) {
+                        sumPointsExerciseGroupsMandatory += exerciseGroup!.exercises![0]!.maxPoints!;
```

Comment:
Is 'sumPointsExerciseGroupsMandatory' defined or initialized with some value before this? we use it later and might need this..

Reply:
Yes in line 58 :)